haiku-apps/memochip/memochip-1.3~git.recipe

```diff
@@ -0,0 +1,43 @@
+SUMMARY="A simple desktop memo tool"
+DESCRIPTION="A simple desktop memo tool, similar to post-it notes"
+HOMEPAGE="https://github.com/HaikuArchives/MemoChip"
+COPYRIGHT="2000 Hideki Naito(bo0wy)"
+LICENSE="unknown"
+REVISION="1"
+srcGitRev="f7d08de2d71f9d73ea89aeda627fd24871ce3512"
+SOURCE_URI="$HOMEPAGE/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="14d0bb56c469e369b553e2b61a78f8c7597198a9f218e242f8468b8140d46710"
+SOURCE_FILENAME="MemoChip-$portVersion-$srcGitRev.tar.gz"
+SOURCE_DIR="MemoChip-$srcGitRev"
+
+ARCHITECTURES="?x86_gcc2 x86_64"
+
+PROVIDES="
+	MemoChip = $portVersion
+	app:MemoChip = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	makefile_engine
+	cmd:g++
+	cmd:make
+	"
+
+BUILD()
+{
+	make $jobArgs OBJ_DIR=objects
+}
+
+INSTALL()
+{
+	install -d $appsDir
+	install -t $appsDir -m755 objects/MemoChip
+
+	addAppDeskbarSymlink $appsDir/MemoChip MemoChip
```

Comment:
```diff
-	addAppDeskbarSymlink $appsDir/MemoChip MemoChip
+	addAppDeskbarSymlink $appsDir/MemoChip
```
(The last paramter is not required, since we are using the same name.)